pages/common/git-rebase.md

```diff
@@ -1,23 +1,24 @@
 # git rebase
 
 > Apply local commits on top of another branch's history.
```

Comment:
Maybe "Reapply the commits from one branch to the tip of another branch" would be better?